api/numaresourcesoperator/v1/helper/nodegroup/nodegroup.go

```diff
@@ -58,7 +58,7 @@ func (ttr Tree) Clone() Tree {
 // Unfortunately this is with very, very high probability a refactoring mistake which slipped in unchecked.
 // One of the key design assumptions in NROP is the 1:1 mapping between NodeGroups and MCPs.
 // This historical accident should be fixed in future versions.
-func FindTrees(mcps *mcov1.MachineConfigPoolList, nodeGroups []nropv1.NodeGroup) ([]Tree, error) {
+func FindTreesOpenshift(mcps *mcov1.MachineConfigPoolList, nodeGroups []nropv1.NodeGroup) ([]Tree, error) {
```

Comment:
we need to have
- a `FindTrees` function/alias pointing to `FindTreesOpenshift`
- a comment documenting the alias is deprecated, use platform-specific function instead

Reply:
updated, thanks